database/schema/init_schema.sh

```diff
@@ -23,3 +23,9 @@ else
     echo "Schema initialization skipped."
 fi
 
+echo "Enabling extensions."
+psql -c "CREATE EXTENSION pg_repack" -d ${POSTGRESQL_DATABASE}
+psql -c "ALTER DEFAULT PRIVILEGES IN SCHEMA repack GRANT INSERT ON TABLES TO PUBLIC" -d ${POSTGRESQL_DATABASE}
+psql -c "ALTER DEFAULT PRIVILEGES IN SCHEMA repack GRANT USAGE, SELECT ON SEQUENCES TO PUBLIC" -d ${POSTGRESQL_DATABASE}
+psql -c "ALTER USER ve_db_user_manager WITH SUPERUSER" -d ${POSTGRESQL_DATABASE}
```

Comment:
I wouldn't do that. This will create difference between `ve_db_user_manager` in local and other environments and can lead to future errors (e.g. manager API working locally but not in prod). Let's use only `ve_db_admin` (and his equivalent in envs using RDS db) as user with superpermissions.

(Another alternative would be to create new user with super permissions, used exclusively for manager-admin but I'm not sure if such permission can be granted in RDS to regular users.)